src/styles/main.scss

```diff
@@ -1,3 +1,71 @@
+@use 'sass:list';
+
 body {
   margin: 0;
+  font-family: Arial, Helvetica, sans-serif;
 }
+
+.calendar {
+  --cell-size: 100px;
+  --gap-size: 1px;
+  --padding: 10px;
+
+  $days: mon, tue, wed, thu, fri, sat, sun;
```

Comment:
You can add some value to each key of array
https://sass-lang.com/documentation/at-rules/control/each/#with-maps